cli/pcluster/easyconfig.py

```diff
@@ -29,6 +29,7 @@
 
 from . import cfnconfig
 from pcluster.utils import get_supported_schedulers
+from pcluster.utils import get_supported_os
```

Comment:
you can write: `from pcluster.utils import get_supported_os, get_supported_schedulers`